scripts/stubtest/allowlist.txt

```diff
@@ -393,6 +393,10 @@ django.urls.resolvers.URLPattern.lookup_str
 django.urls.resolvers.URLResolver.url_patterns
 django.urls.resolvers.URLResolver.urlconf_module
 django.utils.connection.BaseConnectionHandler.settings
+django.db.models.Q.referenced_base_fields
```

Comment:
We also have to figure out a way to work with `cached_property` without this hack

Reply:
I think Django was planning to replace their homegrown `cached_property` implementation with `functools.cached_property` in 5.1 or 5.2.

If we have the patience to wait until then, this will resolve itself 😄

Reply:
I think it might take quite some time? According to this comment and ticket https://code.djangoproject.com/ticket/30949#comment:14

Or is it mentioned to be replaced someplace else?

Reply:
No, I think you're right, it will take years.